commit 28ff4fda9e5b4917375a49a3c6f76aed0cdd55ae
Author: Adrien Schildknecht <adrien+dev@schischi.me>
Date:   Tue Nov 10 20:12:19 2015 +0100

    mmc: kconfig: replace FAULT_INJECTION with FAULT_INJECTION_DEBUG_FS
    
    Fault-injection capability for MMC IO uses debugfs entries to configure
    the attributes.
    FAULT_INJECTION_DEBUG_FS must be enabled to use FAIL_MMC_REQUEST.
    
    Replace FAULT_INJECTION with FAULT_INJECTION_DEBUG_FS.
    Also remove 'select DEBUG_FS' since FAULT_INJECTION_DEBUG_FS depends on
    it.
    
    Signed-off-by: Adrien Schildknecht <adrien+dev@schischi.me>
    Signed-off-by: Ulf Hansson <ulf.hansson@linaro.org>

diff --git a/lib/Kconfig.debug b/lib/Kconfig.debug
index 8c15b29d5adc..cdbf7fd69eb9 100644
--- a/lib/Kconfig.debug
+++ b/lib/Kconfig.debug
@@ -1523,8 +1523,7 @@ config FAIL_IO_TIMEOUT
 
 config FAIL_MMC_REQUEST
 	bool "Fault-injection capability for MMC IO"
-	select DEBUG_FS
-	depends on FAULT_INJECTION && MMC
+	depends on FAULT_INJECTION_DEBUG_FS && MMC
 	help
 	  Provide fault-injection capability for MMC IO.
 	  This will make the mmc core return data errors. This is